app/models/bot/alegre.rb

```diff
@@ -711,7 +730,7 @@ def self.is_text_too_short?(pm, length_threshold)
     unless pm.alegre_matched_fields.blank?
       fields_size = []
       pm.alegre_matched_fields.uniq.each do |field|
-        fields_size << pm.send(field).to_s.split(/\s/).length if pm.respond_to?(field)
+        fields_size << self.get_number_of_words(pm.send(field)) if pm.respond_to?(field)
```

Comment:
This is the root of CV2-3476 . I started using `get_number_words` here, but dropped the `.to_s` unintentionally without knowing that this handled the case of `nil`. (i.e., `nil.to_s` generates a blank string). I'll update get_number_of_words to include `.to_s` and return 0 for `nil` input.